api/src/main/java/eu/coatrack/api/Proxy.java

```diff
@@ -67,6 +72,9 @@ public Proxy(String id) {
     @Temporal(javax.persistence.TemporalType.DATE)
     private Date deletedWhen;
 
+    @Enumerated(EnumType.STRING)
+    private ProxyStates status;
```

Comment:
I would suggest to not store this in the database, as it can be easily calculated "on the fly".

Reply:
Then I would need to send a list to Model and View

Reply:
Done